packages/utils/src/url-ops.ts

```diff
@@ -0,0 +1 @@
+export const isExternalLink = (link: string) => /^(http|https|mailto):/i.test(link);
```

Comment:
```suggestion
/** A short description of what it does */
const isExternalLink = (link: string) => /^(http|https|mailto):/i.test(link);
```
@yauheni-deriv Let's add some TSDoc 🙏🏻 It's a bit confusing what `isExternalLink` means based on the regex 🤔